test/session.test.js

```diff
@@ -750,7 +749,10 @@ test('does not clear cookie if no session cookie in request', async t => {
 
 test('only save session when it changes', async t => {
   t.plan(6)
-  const setStub = sinon.stub()
+  let setCount = 0
+  function setStub () {
```

Comment:
I believe it doesn't even need an extra function.

Reply:
removed that extra.